tools/IoTEdgeDevOps/DevOpsLib/BuildManagement.cs

```diff
@@ -72,7 +72,9 @@ public async Task<IList<VstsBuild>> GetBuildsAsync(HashSet<BuildDefinitionId> bu
                 return buildDefinitionIds.Select(i => VstsBuild.CreateBuildWithNoResult(i, branchName)).ToList();
             }
 
-            return JsonConvert.DeserializeObject<VstsBuild[]>(result["value"].ToString()).ToList();
+            IList<VstsBuild> builds = JsonConvert.DeserializeObject<VstsBuild[]>(result["value"].ToString()).ToList();
+
+            return builds;
```

Comment:
I split this because I often go in here and add debug statements. If anyone wants me to restore it I can.